mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterActor.java

```diff
@@ -658,6 +684,26 @@ static class GetRegisteredTaskExecutorsRequest {
         ClusterID clusterID;
     }
 
+    @Value
+    @Builder
+    static class GetActiveJobsRequest {
+        ClusterID clusterID;
+        Optional<Integer> startingIndex;
+        Optional<Integer> pageSize;
+
+        public GetActiveJobsRequest(ClusterID clusterID) {
+            this.clusterID = clusterID;
+            this.pageSize = Optional.empty();
+            this.startingIndex = Optional.empty();
+        }
+
+        public GetActiveJobsRequest(ClusterID clusterID, Optional<Integer> startingIndex, Optional<Integer> pageSize) {
+            this.clusterID = clusterID;
+            this.pageSize = pageSize;
+            this.startingIndex = startingIndex;
+        }
```

Comment:
nit: you don't need to explicitly write this constructor since you have an `@Value` annotation for the class which brings in the `@AllArgsConstructor`.

Reply:
It seems if I don't add it explicitly or use '@allArgsConstructor' the extra ctor broke the '@value' and it won't compile.